pkg/cue/process/handle.go

```diff
@@ -185,12 +185,18 @@ func (ctx *templateContext) BaseContextFile() string {
 	buff += fmt.Sprintf(model.ContextPublishVersion+": \"%s\"\n", ctx.publishVersion)
 
 	if ctx.appLabels != nil {
-		bt, _ := json.Marshal(ctx.appLabels)
+		bt, err := json.Marshal(ctx.appLabels)
+		if err != nil {
+			return fmt.Sprint(err)
```

Comment:
We should add `error` to the return value of the function instead of returning error string.